contrib/debian/patches/README

```diff
@@ -1,3 +0,0 @@
-0xxx: Grabbed from upstream development.
```

Comment:
Non-blocking: We could leave this file in, since it could be relevant in future.